### PR TITLE
refactor: merge speaker members

### PR DESCRIPTION
**Pull request by Prerna290**
## Type
- ( ) Fix
- (X) Refactor
- ( ) Style
- ( ) Docs
- ( ) Feat
- ( ) Hotfix

## Screenshots
image
image

## Bundle Size
3426.27 KB

### CODE REVIEW

**Comment by arpansac** [comment]
Change as per discussion @Prerna290

File: apps/commudle-admin/src/app/feature-modules/public-community/components/members/members.component.html [hunk 1]
```diff
@@ -1,14 +1,33 @@
...
+            [alignFollowToRight]="true"
+          ></app-user-profile-card-large>
+        </ng-container>
+      </div>
 
-    <div *ngIf="canLoadMore" class="load-more">
-      <button (click)="getMembers()" nbButton outline size="small" status="info">Load More...</button>
+      <div *ngIf="canLoadMore" class="load-more">
+        <button (click)="getMembers()" nbButton outline size="small" status="info">Load More</button>
+      </div>
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/public-community/components/members/members.component.html]: <nb-card>
    <nb-card-body>
   </nb-card-body>
</nb-card>

maintain this hierarchy while using nb-card

File: apps/commudle-admin/src/app/feature-modules/public-community/components/members/members.component.ts [hunk 2]
```diff
@@ -33,7 +39,9 @@ export class MembersComponent implements OnInit, OnDestroy {
         this.community = data.community;
         if (this.community) {
           this.getMembers();
+          this.getSpeakerDetails();
           this.seoService.setTitle(`Members | ${this.community.name}`);
+          this.seoService.setTitle(`Speakers | ${this.community.name}`);
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/public-community/components/members/members.component.ts]: this will not work?

**Review by ArshdeepGrover — CHANGES_REQUESTED** [review]

File: apps/commudle-admin/src/app/feature-modules/public-community/components/members/members.component.html (hunk 1 above)
Comment (Prerna290) [reply apps/commudle-admin/src/app/feature-modules/public-community/components/members/members.component.html]: Done

**Comment by Prerna290** [comment]
image

**Review by ArshdeepGrover — APPROVED** [review]

**Pull request merged**